### DISCUSSION TITLE
Why i can't use deleted organization name as my new username account

### DISCUSSION THREAD

**Discussion by kykurniawan** [npm/feedback#670] [Registry] [answered]
Hello, why can't I reuse the organization username I deleted for my new npmjs account?  when i try to register they show an error that the scope name is already in use.  even though after I make sure there were no personal or organizational accounts that used that username.

**Comment by monishcm** [comment, marked as answer]
Thanks for the feedback. I've added this to the backlog for investigation. We shall contact you if we need more information. If you need this resolution for a specific username, try contacting the support if they can help you.

**Comment by kamcpp** [comment]
Hi, I have exactly the same issue. I cannot create a new org with the same name as the org I've removed recently.

**Reply by kykurniawan** [reply]
Hi, my issue has been resolved by the support team, I contacted them and said what happened, they helped restore the username I accidentally deleted.

**Reply by simpleneeraj** [reply]
Same problem here and I don't get any solution yet, how I can recover my old username account

**Reply by kykurniawan** [reply]
> Same problem here and I don't get any solution yet, how I can recover my old username account 

Have you tried contacting the support team?  if not, try contacting them, they will help you.
